src/Symfony/Component/Workflow/Exception/BlockedTransitionException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Exception;
+
+use Symfony\Component\Workflow\TransitionBlockerList;
+
+/**
+ * Thrown by Workflow when a transition is applied on a subject that is
```

Comment:
> Thrown by the workflow when a transition is not enabled.

(I'm not a native English speaker, but it seems easier to understand)

Reply:
I agree. I just wanted to use the vocabulary set out by the `Workflow` component. 

I'll use the sentence you suggested.